tests/scalability/generate_brac_data/generate-brac-data.js

```diff
@@ -0,0 +1,203 @@
+const bracPlaceFactory = require('../../factories/brac/contacts/brac-place');
+const bracPersonFactory = require('../../factories/brac/contacts/brac-person');
+const bracUserFactory = require('../../factories/brac/users/brac-user');
+const bracSurvey = require('../../factories/brac/reports/brac-survey');
+const fs = require('fs');
+const path = require('path');
+const dataConfig = require('./data-config.json');
+const sizeConfig = require('./size-config.json');
+const createCsvWriter = require('csv-writer').createObjectCsvWriter;
+const args = process.argv.slice(2);
+const districtHospitalName = args[0] + '-districthospital';
+const numberOfDistrictHospitals = sizeConfig.number_of_district_hospitals;
+const numberOfManagersPerDistrictHospital = sizeConfig.number_of_managers_per_district_hospitals;
+const numberOfHealthCentersPerDistrictHospital = sizeConfig.number_of_health_centers_per_district_hospital;
+const numberOfChwPerHealthCenter = sizeConfig.number_of_chw_per_health_center;
+const numberOfClinicsPerHealthCenter = sizeConfig.number_of_clinics_per_health_center;
+const numberOfFamilyMembers = sizeConfig.number_of_family_members;
+const dataDirectory = args[1];
```

Comment:
Something is amiss here. This file is called like: `node generate-brac-data.j [data-directory-path]` (As per the readme), so we never have two arguments.

Is the first argument supposed to be some name? 
Right now, if I follow the readme I get this error: 
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
```

Also, I think we should add a nice check for the existence of the data directory passed to the script. Right now, if I pass a nonexistent path, I get weird errors about not being able to create other directories.

Reply:
I actually added 2 more parameters: name and instance.
`generate-brac-data.js [district-hospital-base-name] [data-directory-path] [instance]`
**district-hospital-base-name:** is going to be used as a name base to generate all the hierarchy, this having in mind we can parallelize the execution (please read comment about the limitations of this feature right now).
**instance:** is going to be used to generate a config.json file for the next step script, with a url to the instance and a list of generated users info.

- [X] Added the check for the existence of the data directory.
- [X]  README.md updated